tasks/bot-avatars.md

```diff
@@ -131,7 +131,7 @@ on things from this list.  Be creative!
 - sales bot (bot that plugs into your Etsy/Squarespace/Shopfiy/Stripe/Square/etc
   eCommerce site or payments system and updates the chat every time an item is
   sold or refunded)
-- revenue bot (bots that plug into your eCommerce site and updates the chat on
+- revenue bot (bots that plugs into your eCommerce site and updates the chat on
```

Comment:
But you're right that there's something broken in the grammar of this line! To fix subject-verb agreement in the whole line, either change both the verbs:

> revenue bot (bots that plug into your eCommerce site and update the chat on

OR, change the subject to be singular:

> revenue bot (bot that plugs into your eCommerce site and updates the chat on